webpack-demo/src/modules/task.js

```diff
@@ -0,0 +1,7 @@
+export default class Task {
```

Comment:
- [ ] Based on project requirements, you're to create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
- description [string].
- completed [bool].
- index: [number].
- Then Write a function to iterate over the tasks array and populate an HTML list item element for each task 
This will be a static array object. Kindly stick to that requirements ✅